ipn/store/awsstore/store_aws_stub.go

```diff
@@ -13,6 +13,6 @@ import (
 	"tailscale.com/types/logger"
 )
 
-func New(logger.Logf, string) (ipn.StateStore, error) {
+func New(_ logger.Logf, _ string, _ string) (ipn.StateStore, error) {
```

Comment:
why the underscores?

Reply:
I figured with this being a stub it didn't need named inputs, but I was being lazy. Fixed